### DISCUSSION TITLE
No Sidebar ????

### DISCUSSION THREAD

**Discussion by titoff002** [themeselection/flyonui#23] [Q&A] [answered]
Hello,

First of all, very good work!

I have a question: there is a "navbar" component but I couldn't find a **"SideBar"** component. Are you going to add one? Otherwise, would you have any code/tips to give to create a “sidebar” like the one on your institutional site?

Thanks for your help.

Good day.

**Comment by PruthviPraj00** [comment]
Hello @titoff002, 

Thank you for your kind words and support!  

Currently, we don't have a dedicated "SideBar" component, but you can easily create similar functionality using our [Drawer with navigation](https://flyonui.com/docs/overlays/drawer/#drawer-with-navigation). It’s a versatile option that works great for implementing a sidebar-style layout.  

Feel free to check it out and let us know if you need any further assistance!  

Have a great day!

**Comment by PruthviPraj00** [comment, marked as answer]
Hello @titoff002,  

Starting from version 1.2.0, FlyonUI includes a sidebar component. Feel free to check it out!  

Thank you for your continued support of FlyonUI!

**Comment by bessone** [comment]
Hello,

is there a possibility to do, or maybe to have directly in the future, the collapsible sidebar, where only the icons remain visible and the sidebar expands when the mouse passes over it?
